ui/components/app/nfts-items/nfts-items.js

```diff
@@ -181,34 +191,79 @@ export default function NftsItems({
                   key={`nft-${i}`}
                   className="nfts-items__item-wrapper"
                 >
-                  <Card
-                    padding={0}
-                    justifyContent={JustifyContent.center}
-                    className="nfts-items__item-wrapper__card"
-                  >
-                    {nftImage ? (
-                      <button
-                        className="nfts-items__item"
-                        style={{
-                          backgroundColor,
-                        }}
-                        onClick={handleImageClick}
-                      >
-                        <img
-                          className="nfts-items__item-image"
-                          data-testid="nft-image"
-                          src={nftImage}
-                          alt={nftImageAlt}
+                  {process.env.MULTICHAIN ? (
+                    <BadgeWrapper
+                      badge={
+                        <AvatarNetwork
+                          size={Size.SM}
+                          name={currentChain?.nickname}
+                          src={currentChain?.rpcPrefs?.imageUrl}
+                          borderColor={BorderColor.borderMuted}
+                          data-testid="nft-network-badge"
+                        />
+                      }
+                      anchorElementShape={
+                        BadgeWrapperAnchorElementShape.rectangular
+                      }
+                      positionObj={{ top: -2, right: -2 }}
```

Comment:
Bumping these to `4` may allow for the border a bit more. I checked Figma and there is a 4px offset
```suggestion
                      positionObj={{ top: -4, right: -4 }}
```